### DISCUSSION TITLE
Difference between `subscribe()` and `channel()`

### DISCUSSION THREAD

**Discussion by nathantew14** [supabase/supabase#18147] [Questions] [answered]
The [realtime docs overview](https://supabase.com/docs/guides/realtime) shows this method to subscribe to changes:
```
// Initialize the JS client
import { createClient } from '@supabase/supabase-js'
const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY)

// Create a function to handle inserts
const handleInserts = (payload) => {
  console.log('Change received!', payload)
}

// Listen to inserts
const { data: todos, error } = await supabase.from('todos').on('INSERT', handleInserts).subscribe()
```

However, in the [usage guide for Postgres Changes](https://supabase.com/docs/guides/realtime/postgres-changes#usage), there is a completely different method:
```
supabase
    .channel('schema-db-changes')
    .on(
      'postgres_changes',
      {
        event: '*',
        schema: 'public',
      },
      (payload) => console.log(payload)
    )
    .subscribe()
```

And in fact, I can't seem to find the first method anywhere else. How are they different? When should each be used? Thanks.

**Comment by GaryAustin1** [comment, marked as answer]
This is a duplicate of a question asked in Discord and the first example is outdated from a previous version of Realtime.  The docs need to be updated in that section.
